Makefile

```diff
@@ -8,6 +8,7 @@ INSTALL_PATH ?= $(CURDIR)
 #-----------------------------------------------
```

Comment:
Since you've made substantive changes, it might also be a good time to update the copyright notice. Or maybe not. I'm not sure about Basho's policies on such things.